core/core.go

```diff
@@ -427,6 +427,21 @@ func (c *Core) RenameSCFolder(tempFolderPath string, smartContractName string) (
 	}
 	return scFolderName, err
 }
+func (c *Core) Foldercheck(smartContractName string) interface{} {
+    folderName := c.cfg.DirPath + "SmartContract/" + smartContractName
+    info, err := os.Stat(folderName)
+    if err != nil {
+        c.log.Error("Unable to get the file info, error is: ", err)
```

Comment:
No need for the Error logger to be added here, since the necessary logs will be present outside of this function